README.md

```diff
@@ -45,17 +45,20 @@ The details of how to build it are described below (easy to follow).
 1. Operating system and tools
 
    As of January 2019 (Horizen v2.0.16) this program supports Linux, macOS Sierra/High Sierra and Windows.
-   The Linux tools you need to build and run the Wallet GUI are Git, Java (JDK8) and
+   The Linux tools you need to build and run the Wallet GUI are Git, Java (JDK12) and
    Ant. If using Ubuntu Linux, they may be installed via command: 
    ```
-   user@ubuntu:~/build-dir$ sudo apt-get install git default-jdk ant
+   user@ubuntu:~/build-dir$ sudo apt update
+   user@ubuntu:~/build-dir$ sudo add-apt-repository ppa:linuxuprising/java
+   user@ubuntu:~/build-dir$ sudo apt install oracle-java12-installer
```

Comment:
With Oracle's new licensing/release model and nearly no differences to openjdk we should install openjdk instead of oraclejdk. https://adoptopenjdk.net/installation.html?variant=openjdk12&jvmVariant=hotspot#linux-pkg is a good resource for this. Or alternatively default-jdk should work just fine.